src/views/AIView.tsx

```diff
@@ -27,9 +27,12 @@ export function AIView({ selection }: { selection: string | null }) {
 
 async function interactWithAi(selection: string) {
   try {
-    const aiReply = await invoke<string>('interact_with_ai', {
-      selection,
-    });
+    const command = Command.sidecar('bin/python/main', ['--text', `${selection}`]);
```

Comment:
@cguedes @gjreda Is the idea here that `main` will eventually be the entrypoint for all Python commands? Or that we'll have separate Python scripts for all commands? I initially misread `bin/python/main` as being the `python` binary itself and started getting worried about security.